packages/wallets/src/components/WalletHistory.tsx

```diff
@@ -249,6 +263,17 @@ export default function WalletHistory(props: Props) {
         metadata={metadata}
         expandedCellShift={1}
         uniqueField="name"
+        onToggleExpand={async (rowId: string, expanded: boolean) => {
+          const memos = (await getTransactionMemo({ transactionId: rowId.split('-')[0] })).data;
```

Comment:
We only want to call this if expanded, right? Otherwise it gets called when expanding or collapsing.

Reply:
You're right. Fixed.